generators/app/index.js

```diff
@@ -25,7 +25,7 @@ module.exports = class extends yeoman {
   }
 
   configuring() {
-    this.log('configuring');
+    // This.log('configuring');
```

Comment:
this change does not seem intentional...

Reply:
removed